components/ai/AIResponseFormatter.tsx

```
@@ -0,0 +1,265 @@
+
+  // For non-structured responses, return formatted text
+  return (
+    <div className="ai-response-text">
+      <div className="whitespace-pre-wrap text-sm leading-relaxed text-gray-700">
+        {response.split('\n').map((line, index) => {
+          // Format bold text
+          const formattedLine = line.replace(/\*\*(.*?)\*\*/g, '<strong>$1</strong>');
+          
+          return (
+            <div key={index} className={line.trim() ? 'mb-2' : 'mb-1'}>
+              <span dangerouslySetInnerHTML={{ __html: formattedLine }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)